src/routes/StyleGuide.jsx

```diff
@@ -117,6 +127,69 @@ const StyleGuide = () => {
           </Paragraph>
         </Modal>
       </Flex>
+
+      <Divider />
+
+      <Flex justify="flex-start" className="px-12">
+        <Heading level={2}>Input</Heading>
+      </Flex>
+      <Flex vertical justify="flex-start" gap="middle" className="px-12">
+        <CustomInput
+          className="w-80"
+          placeholder="Basic Input"
+          allowClear
+          onChange={onChange}
+        />
+
+        <CustomInputWithLabel
+          className="w-80"
+          content="Input with label"
+          placeholder="Input with label"
+        />
+
+        <CustomInputWithPassword
+          className="w-80"
+          placeholder="input password"
+          iconRender={(visible) =>
+            visible ? <EyeTwoTone /> : <EyeInvisibleOutlined />
+          }
+        />
```

Comment:
Please use font specified in the style guide

Reply:
@congruiyS2023 
I don't understand.
This is an icon from AntD. It does not have font.
Also, I did follow the font specified in the style guide in this PR.

Reply:
Click the file name to see full lines of codes, I'm not referring to the icon here. Also, I don't think this is Barlow: 
<img width="332" alt="image" src="https://github.com/congruiyS2023/platemate-web/assets/120247054/6828a7ea-2b60-4917-b37c-f12e00c28761">
This is what Barlow should look like: 
<img width="256" alt="image" src="https://github.com/congruiyS2023/platemate-web/assets/120247054/3ae98611-eeba-4253-8f6d-6ee925396a07">
Notice the bottom right, the font family is not Barlow: 
<img width="1422" alt="image" src="https://github.com/congruiyS2023/platemate-web/assets/120247054/ec2548ac-2131-409d-aa39-1995f1fe288c">

Reply:
just add font-paragraph for Barlow and font-heading for epilogue, it's defined in tailwind.config.js

Reply:
Resolved